assets/src/components/SidebarFields/ImageSidebarField.js

```diff
@@ -3,31 +3,32 @@ import { Button } from '@wordpress/components';
 import { ImageHoverControls } from '../ImageHoverControls';
 import { __ } from '@wordpress/i18n';
 
-export const ImageSidebarField = ({ value, setValue, label }) => (
+export const ImageSidebarField = ( { value, setValue, label } ) => (
   <div className="components-base-control mb-3">
-    <label className="components-base-control__label mb-2 d-block">
-      {label}
+    <label className="components-base-control__label mb-2 d-block" htmlFor="media-elt">
+      { label }
     </label>
     <MediaUploadCheck>
       <MediaUpload
-        title={label}
-        type='image'
-        value={value.id}
-        onSelect={({ id, url }) => setValue(id.toString(), url)}
-        allowedTypes={[ 'image' ]}
-        render={({ open }) => value.url ?
-          <div style={{ position: 'relative' }}>
+        id="media-elt"
```

Comment:
This might cause issues if we have several components like this at the same time, since ids are supposed to be unique

Reply:
Ok, I just left the htmlFor on the label to prevent lint errors.